setup/docker/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM java:8-alpine
+
+MAINTAINER Danilo Recchia <danilo.recchia@vortus.solutions>
+
+RUN apk upgrade --update && \
```

Comment:
Multiples RUN create multiples docker layers. Thats not performatic and not recommended.